test/transaction/vote_cr_tx.lua

```diff
@@ -43,24 +41,22 @@ then
     fee = 0.1
 end
 
-if vote_candidate_vote == 0
+if vote_candidates == ""
 then
-    vote_candidate_vote = 0.1
+    print("candidates is nil, should use --candidates or -cds to set it.")
+    return
 end
 
-if vote_candidate_code == ""
+if vote_candidate_votes == ""
```

Comment:
should be {}

Reply:
should be if next(vote_candidates) == nil